packages/styles/src/tooltip/tooltip.scss

```diff
@@ -0,0 +1,44 @@
+@import "../core/_dev";
+
+.as-tooltip {
+  display: inline-flex;
+  position: relative;
+  padding: get-spacing(1) get-spacing(2);
+  border-radius: get-spacing(1);
+  background: var(--as-color-type-01, #{$color-type-01});
+  color: var(--as-color-type-04, #{$color-type-04});
+  font: $font-body;
+
+  &::after {
+    content: "";
+    position: absolute;
+    border-width: 5px;
+    border-style: solid;
+    border-radius: 2px;
+    border-color: var(--as-color-type-01, #{$color-type-01});
+  }
+
+  &--bot::after {
```

Comment:
maybe we could use `bottom`instead of `bot`?

Reply:
we use bot all along the project

Reply:
so bot-it-is 🤖